fix: Organize-imports can sometimes add unwanted imports

subhash-arabhi opened the pull request:
Changes to fix the [bug](https://github.com/oracle/javavscode/issues/212). This [PR](https://github.com/apache/netbeans/pull/4561) created this side effect. 

### Brief
When there is a name clash in our package with java.lang.*, the addImports function from GeneratorUtilities is adding unwanted package imports. Refer bug for more details

mbien reviewed (commented):
makes sense to me, thanks!

(had a deja-vu since I thought I fixed this already https://github.com/apache/netbeans/pull/7363, but this is a different edge case)

could you squash the two commits?

subhash-arabhi commented:
> makes sense to me, thanks!
> 
> (had a deja-vu since I thought I fixed this already #7363, but this is a different edge case and also a different action: fix imports vs organize imports)
> 
> could you squash the two commits?

@mbien I have squashed the commits, could you approve and merge this PR

mbien reviewed (approved):
thanks for squashing
